defaults/main.yml

```diff
@@ -3,14 +3,26 @@ k8s_kubeconfig: ~/.kube/config
 # <aws|gcp|azure|digitalocean>
 # k8s_cluster_type: ''
 
+# DO NOT edit k8s_aws_load_balancer_type after creating your Service. Delete the
+# nginx-ingress namespace and re-run this role if you really need to change the
+# AWS load balancer type.
+# <nlb|elb>
+k8s_aws_load_balancer_type: elb
```

Comment:
Looking at the rest of the PR, it seems we want to encourage `nlb` use for future projects? If so, should we make this `nlb` here and update the release notes to tell existing users to set theirs to `elb` if they don't want to upgrade?